### PR TITLE
Call api

### PR DESCRIPTION
**Pull request by JDomleo**
Probably a bit shit lol

### CODE REVIEW

File: ShitFood.Web/js/get-restaurants.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+var axios = require('axios');
+
+function getRestaurants() {
+  var cutleryIcon = L.icon({
+    iconUrl: '../img/markers/cutlery.svg',
```
Comment (jamie-lord) [inline ShitFood.Web/js/get-restaurants.js]: This needs to be './img/markers/cutlery.svg' to work

File: ShitFood.Web/js/get-restaurants.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    iconSize: [36, 36], // size of the icon
+    iconAnchor: [0, 0], // point of the icon which will correspond to marker's location
+    popupAnchor: [18, -3] // point from which the popup should open relative to the iconAnchor
+  });
+
+  axios.get('https://shitfoodapi.azurewebsites.net/api/getshit?lat=52.9548&lng=-1.1581')
+      .then(function (response) {
+        for (var i = 0; i < response.length; i++) {
```
Comment (jamie-lord) [inline ShitFood.Web/js/get-restaurants.js]: This needs a minor change as the `response` object isn't just the data you're expecting, it also has some other properties. So the DTO array is actually `response.data`, use that everywhere the the `.then`.

File: ShitFood.Web/js/get-restaurants.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+function getRestaurants() {
+  var cutleryIcon = L.icon({
+    iconUrl: '../img/markers/cutlery.svg',
+    iconSize: [36, 36], // size of the icon
+    iconAnchor: [0, 0], // point of the icon which will correspond to marker's location
+    popupAnchor: [18, -3] // point from which the popup should open relative to the iconAnchor
+  });
+
+  axios.get('https://shitfoodapi.azurewebsites.net/api/getshit?lat=52.9548&lng=-1.1581')
```
Comment (jamie-lord) [inline ShitFood.Web/js/get-restaurants.js]: I've added https://shitfood.wtf in the cors settings so the deployed version will work with the API. When you run the website locally are you using a specific port? I can add that as an exception too so you can get the data locally. Also please could you add a note in the readme about what command to run to minify the js and run the site locally, then we can be sure we're both doing the same thing.

Comment (JDomleo) [reply ShitFood.Web/js/get-restaurants.js]: I'm not using a specific port, added README

File: ShitFood.Web/js/get-restaurants.js (hunk 2 above)
Comment (JDomleo) [reply ShitFood.Web/js/get-restaurants.js]: Ah I didn't realise

File: ShitFood.Web/js/get-restaurants.js (hunk 1 above)
Comment (JDomleo) [reply ShitFood.Web/js/get-restaurants.js]: Nice

**Pull request merged**